### DISCUSSION TITLE
title length incomplete

### DISCUSSION THREAD

**Discussion by lossfunc** [aidenlx/obsidian-zotero#287] [Help]
for long title notes, title name is cut off
<img width="770" alt="image" src="https://github.com/PKM-er/obsidian-zotlit/assets/19942192/79239c39-7fcb-4696-9bd1-23d25831cc95">

**Comment by aidenlx** [comment]
Filenames come from the `filename` template in v2, and the only truncation left is the 255-byte filesystem limit on a single path component — nothing shortens a title beyond that.

The default leads with the citation key, but you can name notes from the title instead:

```liquid
{{ zt.title | default: zt.key }}{% suffix %}
```

Reference: https://zotlit.aidenlx.site/docs/reference/templates/defaults

v2 beta is available now: https://zotlit.aidenlx.site/docs/install-zotlit
Migration guide: https://zotlit.aidenlx.site/docs/how-to/migrate-from-v1
